app/Dockerfile

```diff
@@ -33,7 +36,7 @@ CMD ["npm", "run", "dev"]
 
 # Build the Next.js app
 # =====================================
-FROM base AS builder
+FROM dev-dependencies AS builder
```

Comment:
Whoops, meant to throw that on the release image, not here. I can't add a comment there it looks like since you didn't edit that part. But, this suggestion is for line 61 in the current file

Reply:
Good suggestion, thanks